linking-service/v1/linking-service.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package linking.v1;
+
+option go_package = "github.com/smartcontractkit/chainlink-protos/linking-service/go/v1";
+
+import "google/protobuf/empty.proto";
+import "v1/linking-types.proto";
+
+// ================================================================
+// │  gRPC service interface for Workflow Ownership Proof Service  │
+// ================================================================
+// there are 2 ways to initiate and verify ownership proofs:
+// 1st: signing the transaction via wallet integration (useful for EOAs), wallet calls a proof submit function on the Workflow Registry contract
+// 2nd: signing a transaction via MSIG (using CRE CLI), execution transaction calls a proof submit function on the Workflow Registry contract
+service WorkflowOwnershipProofService {
```

Comment:
I don't think we should be exposing the entire service to nodes - they should not be allowed to initiate linking / unlinking / etc. - imo we should tailor this similarly to how we did with the Credit Reservation client (which exposes only a small subset of the Billing Service)

I assume we only need `ListOwners`, `GetOrganizationFromWorkflowOwner` and `GetOrganizationFromWorkflowID`?